<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>224</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="ColorWidget" name="bgColorWidget" native="true"/>
   </item>
   <item>
    <widget class="KSeparator" name="kseparator"/>
   </item>
   <item>
    <widget class="ColorWidget" name="fgColorWidget" native="true"/>
   </item>
   <item>
    <widget class="KSeparator" name="kseparator_2"/>
   </item>
   <item>
    <widget class="QLabel" name="ratioLabel">
     <property name="text">
      <string notr="true">[Ratio Label]</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="demoLabel">
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="text">
      <string>Hello World</string>
     </property>
     <property name="margin">
      <number>6</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KSeparator</class>
   <extends>QFrame</extends>
   <header>kseparator.h</header>
  </customwidget>
  <customwidget>
   <class>ColorWidget</class>
   <extends>QWidget</extends>
   <header>colorwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
